cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -196,34 +196,64 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
     reconfigure_vm(small_vm, orig_config)
 
 
-@pytest.mark.manual
 @pytest.mark.tier(1)
-@pytest.mark.parametrize('change_type', ['sockets', 'memory'])
-def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
+@pytest.mark.parametrize('disk_type', ['thin', 'thick'])
+@pytest.mark.parametrize(
+    'disk_mode', ['persistent', 'independent_persistent', 'independent_nonpersistent'])
+@pytest.mark.provider([VMwareProvider], override=True)
```

Comment:
Q: Why only this provider? I tested you changes locally with one of the parametrization for RHV and it runs fine, except the cleanup, which fails due to https://bugzilla.redhat.com/show_bug.cgi?id=1692801